src/electionguard-ui/ElectionGuard.UI.Lib/Services/Interfaces/IAuthenticationService.cs

```diff
@@ -1,8 +1,10 @@
-﻿namespace ElectionGuard.UI.Lib.Services;
+﻿using Microsoft.Extensions.Logging;
+
+namespace ElectionGuard.UI.Lib.Services;
 
 public interface IAuthenticationService
 {
-    Task Login(string username);
+    Task Login(string username, ILogger? logger);
```

Comment:
why not pass this into the ctor? it's not a typical pattern do pass logger in like this

Reply:
The Logger is a templated class that is based on the page itself.  Putting it into the ctor of the AuthenticationService uwing DI will not get the specific templated type for that page.

Reply:
Right. It should be ILogger<AuthenticationService>